assets/javascript/dashboard/main.js

```diff
@@ -268,6 +268,15 @@ function dashboard() {
             
             const sanitizedParams = this.sanitizeParams({...this.filters, ...params});
             const urlParams = new URLSearchParams(sanitizedParams);
```

Comment:
don't initialise the params with the values:

```suggestion
            const urlParams = new URLSearchParams();
```